main/main.c

```diff
@@ -153,7 +153,14 @@ void ex2_init(void *pvParameters) {
 
 #ifndef SBAND_IS_STUBBED
     STX_Enable();
-    // PLACEHOLDER: sband hardware init
+    vTaskDelay(ONE_SECOND/10);
```

Comment:
This should all be in an sband_init function in the hardware interface

Reply:
I actually wrote one, if we ever get the over PRs merged ;->